workflows/gtfs_validator_execution.yml

```diff
@@ -0,0 +1,173 @@
+                              Content-Type: text/plain
+                          url: ${resultsEndpoint + jobID + "/execution_result.json"}
+                        result: executionResultResponse
+                    - validateStatus:
+                        next: logExecutionResult
+                # Retrying 120 times with a delay of 10 seconds each time gives 20 minutes wait time.
+                retry:
+                  predicate: ${retry_predicate}
+                  maxAttempts: 120
+                  backoff:
+                    initial_delay: 10
+                    max_delay: 10
+                    multiplier: 1
+      - logExecutionResult:
+          steps:
+          - logExecutionResultMessage:
+              call: sys.log
+              args:
+                  text: 'Execution results:'
+                  severity: INFO
+          - logExecutionResultBody:
+              call: sys.log
+              args:
+                  text: ${executionResultResponse.body}
+                  severity: INFO
+      - checkExecutionResultStatus:
+          steps:
+            - getExecutionStatus:
+                assign:
+                  - isReportGenerated: ${executionResultResponse.body.status == "success"}
+            - validateExecutionStatus:
+                switch:
+                  - condition: ${isReportGenerated}
+                    next: htmlReport
+                next: errorReportGeneration
+      - errorReportGeneration:
+          steps:
+            - logErrorReportGeneration:
+                call: sys.log
+                args:
+                    text: 'Error generating report'
+                    severity: ERROR
+            - returnErrorReportGeneration:
+                return: "Error generating report"
+      - htmlReport:
+          call: googleapis.storage.v1.objects.copy
+          args:
+            destinationBucket: ${datasetsBucket}
+            destinationObject: ${feedID + "%2F" + datasetID + "%2Freport.html"}
+            sourceBucket: ${reportsBucket}
+            sourceObject: ${jobID + "%2Freport.html"}
+            destinationPredefinedAcl: publicRead
+      - logHTMLReportCopied:
+          call: sys.log
+          args:
+              text: ${"HTML report copied to " + datasetsBucket + "/" + feedID + "/" + datasetID + "/report.html"}
+              severity: INFO
+      - jsonReport:
+          call: googleapis.storage.v1.objects.copy
+          args:
+            destinationBucket: ${datasetsBucket}
+            destinationObject: ${feedID + "%2F" + datasetID + "%2Freport.json"}
+            sourceBucket: ${reportsBucket}
+            sourceObject: ${jobID + "%2Freport.json"}
+            destinationPredefinedAcl: publicRead
+      - logJSONReportCopied:
+          call: sys.log
+          args:
+              text: ${"JSON report copied to " + datasetsBucket + "/" + feedID + "/" + datasetID + "/report.json"}
+              severity: INFO
+          next: successfulExecution
```

Comment:
Let's also copy the _system_errors.json_ files that is important in case of failures.

Reply:
Done ✅ 
![image](https://github.com/MobilityData/mobility-feed-api/assets/60586858/5e4a2add-663b-440c-9c71-995a48a2e079)